tests/format/handlebars/block-statement/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -570,6 +624,86 @@ printWidth: 80
   Another thing
 {{~/if~}}
 
+<div>
```

Comment:
You can put this test in a separate dir with small printWidth, instead of deep.nest.